lib/content/message/predictions.ex

```diff
@@ -197,9 +197,7 @@ defmodule Content.Message.Predictions do
 
   defp calculate_crowding_data_confidence(_prediction, _location), do: nil
 
-  defp get_crowding_description([]), do: nil
-
-  defp get_crowding_description(carriage_details) do
+  defp get_crowding_description(carriage_details) when length(carriage_details) == 6 do
```

Comment:
suggestion(non_blocking): it likely doesn't matter in this case, but in general it's good to avoid calculating the full length if you don't need it. Here, you could replace it with:
```suggestion
  defp get_crowding_description([_, _, _, _, _, _]) do
```
For more, see https://www.erlang.org/docs/26/efficiency_guide/commoncaveats.html#length-1

Reply:
The example in the docs works because it can stop after the first 3 elements, but I'd be surprised if this code was able to somehow complete the pattern match without effectively traversing the whole list. Still, I like how the match looks visually in this code. Thanks!

Reply:
This pattern match works the same way: it can stop after 6 elements, where the length() call would need to continue.